Update readme with new process (#37)
nodejs/nodejs-collection: README.md

- Reports on Node.js and JavaScript events (these events must have a code of conduct to be considered)
- Other topics will certainly be considered, but it should be something of interest to the Node.js community

## Editors
We have list of people who review the post before publishing. If the post is specific to some technical topic, we seek review from someone who has knowledge of that domain.

### How to become an editor?

> We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with your details and the team will review it.

Reviewer: What information needs to be included to be an editor?

Author: I think it should include,
- Intro about themself.
- How much time they can give.
- Area of expertise.
- Email address for contact.

What do you suggest?

We have list of people who review the post before publishing. If the post is specific to some technical topic, we seek review from someone who has knowledge of that domain.

### How to become an editor?
We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with these details and the team will review it.
- Name
- Experties (i.e IoT, Node.js Modules, cli, APIs etc)

> - How much time you can give per week. (i.e 1-2 hours)

Reviewer: ```suggestion
Please include the following:
- Name
- Expertise (i.e IoT, Node.js Modules, CLI, APIs etc)
- How much time you can give per week (i.e 1-2 hours)
```

### Benefits of being an editor
- Your name will be added to the [current editors list](https://github.com/nodejs/nodejs-collection#current-editors).
- You'll be added to [editors list on Medium](https://medium.com/the-node-js-collection/about).

> - Possibility to become a member of Node.js Project and avail benefits like [Travel Fund](https://github.com/nodejs/admin#travel-fund).

Reviewer: This is a benefit of being a member of the project.

If being an editor includes being a member of the project, this is inherently true and phrasing it like this is slightly misleading since it's something that comes inherently anyway rather than a specific benefit of this work.

If being an editor does not include being a member of the project, then this is also slightly misleading since there is additional work above and beyond being an editor that this requires.

Author: Being an editor will not include being project member. I think we should keep it nomination based. Like [here](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#collaborator-nominations).
Once we see, someone is actively reviewing posts for 2,3 months, we can nominate them to become part of Node.js Collection ( -> Community Committee -> Node.js Project).

I also don't want to overdo it in the start. Let's go with an easy and simple process, once we have enough editors, we can even put this on hold.

Point to mention Travel Fund here was that people know what benefits would they get in return. People who are not part of the project already, may not even know that there is a travel fund benefit.

Author: In the next PR, I'll introduce a custom logger as discussed in slack. For this PR, ignore it.

Reviewer: >Point to mention Travel Fund here was that people know what benefits would they get in return.

This is a good thing to include for the relevant groups. I do think it's important though to include it in the correct place, and, as you said, this is not something that people _will_ get but something they _could_ get if they stick around. 

>Once we see, someone is actively reviewing posts for 2,3 months, we can nominate them to become part of Node.js Collection ( -> Community Committee -> Node.js Project).

Is this documented anywhere? If so, I think that is the most appropriate place to include that context since it's the context in which it is accurate.

We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with these details and the team will review it.
- Name
- Experties (i.e IoT, Node.js Modules, cli, APIs etc)
- How much time you can give per week (i.e 1-2 hours)

> - Email address

Reviewer: ```suggestion
Please include the following:
- Name
- Expertise (i.e IoT, Node.js Modules, CLI, APIs etc)
- How much time you can give per week (i.e 1-2 hours)
- Email address
```

Author: Done.

If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.

## Resources

> Trello Board [https://trello.com/b/M9VUQl9S/nodejs-collection](https://trello.com/b/M9VUQl9S/nodejs-collection)

Reviewer: ```suggestion

* [Trello Board](https://trello.com/b/M9VUQl9S/nodejs-collection)
```

If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.

## Resources

> Trello Board [https://trello.com/b/M9VUQl9S/nodejs-collection](https://trello.com/b/M9VUQl9S/nodejs-collection)

Reviewer: Board not found.

Author: @RRomoff to make it public

### Key Roles:
- **Contributor/Writer** : Person submitting article or idea for inclusion on the Node.js Collection

> - **Editor** : Technical Reviewer. Person who ensure the content is technically sound/ appropriate.

Reviewer 2: ```suggestion
- **Editor** : Technical Reviewer. Person who ensures the content's technical integrity.
```

## Guidelines/General Tips of Submissions

- Posts from members and community members must be vendor-neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers.

> - Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company.

Reviewer: I'm not sure if this is actually valuable in practice. Have we had any posts that are like this?

Author: @bnb I didn't get what you mean? Can you explain a bit?
To me, "Advertisement for your services or company" is not the best. So, it'd be good to keep this line.

- Your post must be your original content, but can be published on the Node.js Foundation Medium publication, if you received permission from the editors of the website to do so.
- All content should have a byline (preferably by a company engineer) and be published with Creative Commons with Attribution, so you’re welcome to republish on your blog.
- The most interesting posts are ones that teach or explain how to do something in a way maybe others haven’t thought of. Good blog posts show hurdles that were encountered and explain how they were overcome (not that everything is rainbows and unicorns).
- A few posts that have performed well on the Node.js Foundation Medium page include Walmart Strives to be an Online Retailer with Node.js (use case); Node.js Transitions to LTS (news); The Progress of Node.js Post io.js and Node.js merger (progress update).

> - When showing upstreaming of a patch fixing an issue for others, link back to the Github issue so readers can follow along.

Reviewer: Is there a case where this line has actually been used? If not, perhaps worth stiriking it entirely?

Additionally, if we don't strike it, it should be updated to `GitHub`

Author: I'll fix it as `GitHub`

Reviewer: This was resolved but not fixed 😅

Author: Done

- We don’t avoid critical commentary or broad issues but approach them with sensitivity, professionalism, and tact in a way that is beneficial and positive for the community.

> - Node.js Foundation Medium posts should be no longer than 1,000 words and no shorter than 300 words.

Reviewer: ```suggestion
- Node.js Collection posts should be no longer than 1,000 words and no shorter than 300 words.
```

Author: 👍

- Diagrams or photos are strongly encouraged.

## How to submit for consideration

- Please submit a brief summary and the topic of the post to `medium@nodejs.org` for consideration.

> - Members of the Node.js Collection Technical Review Team and Node.js ProjectPR team will review your submission in a timely manner and provide the green light to draft the entire article or provide feedback or direction.

Reviewer: AFAIK this should be:

```suggestion
- Members of the Node.js Collection Technical Review Team and OpenJS Foundation PR team will review your submission in a timely manner and provide the green light to draft the entire article or provide feedback or direction.
```

Author: 👍

- If you are submitting an article or presentation that already exists, please send it in its entirety with a note on the expressed permission to publish from the owner of the content.
- Once your submission has been approved, it will be added to the Node.js Foundation Medium publication publishing calendar and a publish date will be provided, so you may plan to promote accordingly through your personal and company’s social media channels.

## Review Process for Original Posts
- Writers submit an abstract via email at medium@nodejs.org.

- Editors have one week to provide approval or feedback - this is done internally on the alias and not to be shared with the writer until a final decision and suggested changes are reached as a group.

> - If the abstract is approved, the writer will be informed and a task will be created on the [Trello board](https://trello.com/b/M9VUQl9S/nodejs-collection) for the post

Reviewer: 404 Trello board and missing period at the end.

- The contributor will be made a “Writer” on the Node.js Collection medium account.

> - The contributor will write the first draft and Write will submit this draft on Medium and an editor or two will be assigned to review. The contributor will ping the medium alias to alert the editors a draft has been submitted. Clear directions on how to submit will be provided to the contributor.

Reviewer: > The contributor will write the first draft and Write will 

Not quite correctly phrased. `Write` is an error.

Author: -> writer

> - Editor from the Node.js Collection will review the post and provide the writer comments or feedback. This process can take up to a month to complete.

Reviewer: How is this substantially different from the 1 week period stated on line 57?

Author: Line 57 is first approval if it is good enough to include or not.
If it is approved, then we start the review process, which can take up to a month.

- The writer to approve all content and send a note to medium@nodejs.org once finished.

- Once the piece is complete, and the writer approves the content, an editor from Node.js PR team to review one final time and accept into publication.

> - Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/ promo.

Reviewer: Additional unnecessary space after the slash in this line (`planning/ promo.`)

Reviewer: ```suggestion
- Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/promo.
```

- The writer will be informed about the approval and publication date.

## Review Process for Existing Created Medium Posts (to be republished with permission on the Node.js Collection)

> - Members from the evangelism working group are invited to become editors on Medium. If interested, contact here: medium@nodejs.org

Reviewer: There is no such thing as the Evangelism WG. This line should likely be deleted, as it doesn't seem to be relevant to the heading.

Author: I will update it as "Anyone is invited to ......"
Later, we can limit it to CommComm and TSC if needed.

Ok?

Reviewer: SGTM

- Editors are allowed to request to put already available public content on the Node.js Foundation Medium publication if they follow the above guidelines pertaining to “Quality” and “Guidelines/General Tips of Submissions”.

- A writer can send an existing article for consideration to the medium alias.

- The editors decide if the content is acceptable for publication.

> - If accepted, a task will be created in the [Trello board](https://trello.com/b/M9VUQl9S/nodejs-collection) and an editor will be assigned to review the content.

Reviewer: 404 Trello Board.
